build/style.css

```diff
@@ -1,106 +1,4 @@
-header {
-  background-color: #036;
-  border-bottom: 2px solid #fcba19;
-  padding: 0 65px 0 65px;
-  color: #fff;
-  display: flex;
-  height: 65px;
-  top: 0px;
-  position: fixed;
-  width: 100%;
-}
-
-header h1 {
-  font-family: ‘Noto Sans’, Verdana, Arial, sans-serif;
-  font-weight: normal;  /* 400 */
-  margin: 5px 5px 0 18px;
-  visibility: hidden;
-}
-
-header .banner {
-  display: flex;
-  justify-content: flex-start;
-  align-items: center;
-  margin: 0 10px 0 0;
-  /* border-style: dotted;
-  border-width: 1px;
-  border-color: lightgrey; */
-}
-
-header .other {
-  display: flex;
-  flex-grow: 1;
-  /* border-style: dotted;
-  border-width: 1px;
-  border-color: lightgrey; */
-}
-
-:focus {
-  outline: 4px solid #3B99FC;
-  outline-offset: 1px;
-}
-
-/*
-  These are sample media queries only. Media queries are quite subjective
-  but, in general, should be made for the three different classes of screen
-  size: phone, tablet, full.
-*/
-
-@media screen and (min-width: 600px) and (max-width: 899px) {
-  header h1 {
-    font-size: calc(7px + 2.2vw);
-    visibility: visible;
-  }
-}
-
-@media screen and (min-width: 900px) {
-  header h1 {
-    font-size: 2.0em;
-    visibility: visible;
-  }
-}
-
-footer {
-  background-color: #036;
-  border-top: 2px solid #fcba19;
-  color: #fff;
-  font-family: ‘Noto Sans’, Verdana, Arial, sans-serif; 
-}
-
-footer .container {
-  display: flex;
-  justify-content: center;
-  flex-direction: column;
-  text-align: center;
-  height: 46px;
-}
-
-footer ul {
-  display: flex;
-  flex-direction: row;
-  flex-wrap: wrap;
-  margin: 0;
-  color: #fff;
-  list-style: none;
-  align-items: center;
-  height: 100%;
-}
-
-footer ul li a {
-  font-size: 0.813em;
-  font-weight: normal;  /* 400 */
-  color: #fff;
-  border-right: 1px solid #4b5e7e;
-  padding-left: 5px;
-  padding-right: 5px;
-}
-
-a:hover {
-  color: #fff;
-  text-decoration: underline;
-}
-
-:focus {
-  outline: 4px solid #3B99FC;
-  outline-offset: 1px;
+.workloadSummaryCell{
```

Comment:
`.workloadSummaryCell{` => `.workload-summary-cell {`

Reply:
css is typically hyphenated and also be mindful of good linting *slaps wrist*

Reply:
renamed in css and template.